### ISSUE TITLE
Create an application

### ISSUE THREAD

**Issue by Sveinki10** [algolia/cli#13]
Is it on the roadmap for this project to make a command to create an Algolia application from the CLI?

**Comment by khalidelassaad** [comment]
Yes @Sveinki10 ! We'd like to support instantiating a front-end app from the command line for easy quickstart!

This is not planned for our GA release, but for a follow-up release

Great question! 😄

**Comment by keybits** [comment]
Creating indices would also be helpful 😄

**Comment by kai687** [comment]
> Creating indices would also be helpful 😄

@keybits If you add objects to a non-existing index, it'll be created, just like when using the API or API clients. For example: 

```
algolia objects import <new_index> --file records.ndjson
```

**Comment by keybits** [comment]
Aha - perfect, thanks!

**Comment by gregfenton** [comment]
> If you add objects to a non-existing index, it'll be created, just like when using the API or API clients. For example:
> 
> ```
> algolia objects import <new_index> --file records.ndjson
> ```

This would be useful to have stated in the documentation.  If it is there...it isn't obvious.

To me, creating indexes would be a primary use case so it would be great to have that "pop out" in the documentation.  Heck, even having an alias for `algolia indices create ...` to actually run `algolia objects import ...` would be huge in terms of discoverability.

**Comment by gregfenton** [comment]
Oh, and +1 for the ability to create applications.

I'm sure this will involve a decent amount of development in order to authenticate to the Algolia account which is a lot more than simply having some API keys for an existing application, but it would be a fantastic utility to streamline/automate the creation of new instances of applications.  Having to do manual steps dramatically slows things down and increases risk of human error.
